src/renderers/shared/ReactDebugTool.js

```diff
@@ -192,6 +192,63 @@ function resumeCurrentLifeCycleTimer() {
   currentTimerType = timerType;
 }
 
+var lastMarkTimeStamp = null;
+var canUsePerformanceMeasure =
+  typeof performance !== 'undefined' &&
+  typeof performance.mark === 'function' &&
+  typeof performance.clearMarks === 'function' &&
+  typeof performance.measure === 'function' &&
+  typeof performance.clearMeasures === 'function';
+
+function shouldMark(debugID) {
+  if (!isProfiling || !canUsePerformanceMeasure) {
```

Comment:
Not sure if its a overkill. But wanted to confirm if there is any weird behaviour with any browsers. 

Totally agree on that point 👍